smarts/core/waymo_map.py

```diff
@@ -47,19 +59,12 @@
 )
 from smarts.waymo.waymo_utils import WaymoDatasetError
 
-from .coordinates import BoundingBox, Heading, Point, Pose, RefLinePoint
-from .lanepoints import LanePoints, LinkedLanePoint
-from .road_map import RoadMap, RoadMapWithCaches, Waypoint
-from .route_cache import RouteWithCache
-from .utils.file import read_tfrecord_file
-from .utils.geometry import buffered_shape
-from .utils.math import (
-    inplace_unwrap,
-    line_intersect_vectorized,
-    radians_to_vec,
-    ray_boundary_intersect,
-    vec_2d,
-)
+try:
+    import rtree
+except:
+    raise ImportError(
+        "You may not have installed the [argoverse] dependencies required for using Argoverse 2 maps with SMARTS. Install it first using the command `pip install -e .[argoverse]` at the source directory."
```

Comment:
Error message indicates Argoverse 2 maps, but file is waymo_map.py.

Reply:
Good catch, fixed.